spec/models/users_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'John', posts_counter: 30, photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.') }
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter should be present' do
+    subject.posts_counter = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter should be integer' do
+    subject.posts_counter = 'a'
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter should be greater than or equal to zero' do
+    subject.posts_counter = -1
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ Required ] I think it would be nice if you fix these tests, currently, when I run the tests in this file I get the errors shown below. Please see the screenshot below and rectify this.

## Error when running tests in users_spec.rb
![Screenshot from 2023-11-04 22-33-15](https://github.com/sheezu900/blog-rails/assets/16392046/6d7b2888-9194-4382-9538-10a1bc8bed77)